app4java/src/main/java/com/nyxei/pollingus/service/Response.java

```diff
@@ -0,0 +1,24 @@
+package com.nyxei.pollingus.service;
+
+public class Response {
+
+    public String userResponse;
+    private static final long serialVersionUID = 1L;
+
+
+    public Response(String r)
+    {
+        this.userResponse = r;
+    }
+
+    public String getUserResponse()
+    {
+        return this.userResponse;
+    }
+
+    // ajoute la réponse à la liste de réponses de Question
+    public void addResponseToQuestion(IQuestion q)
```

Comment:
Bad design for this method. You say you add something but we can't see what is added.